test/spec.index.js

```diff
@@ -672,6 +710,18 @@ describe('Template Mixins', function () {
                 }));
             });
 
+            it('uses locales translation for hint if a field value isn\'t provided', function () {
+                translate = sinon.stub().withArgs('fields.field-name.hint').returns('Field hint');
+                middleware = mixins({
+                    'field-name': {}
+                }, { translate: translate });
+                middleware(req, res, next);
+                res.locals['radio-group']().call(res.locals, 'field-name');
```

Comment:
Good point. Removed the duplicate.

It does test for the behaviour you were describing though doesn't it?